qanta/wikipedia/cached_wikipedia.py

```diff
@@ -145,6 +145,9 @@ def __getitem__(self, key: str):
             except ImportError:
                 print("Error importing %s" % key)
                 page = None
+            except ValueError:
+                print("Error importing %s" % key)
+                page = None
```

Comment:
On master seems like keys are loading fine. With this change does the wiki cache load correctly? I would be slightly concerned that this would possibly make for an empty wiki cache